content/projects/linux/beginner/_index.md

```diff
@@ -81,3 +81,26 @@ Note: for each instruction write a script and save it under this task
 #### Resources
 
 1. https://www.lifewire.com/beginners-guide-to-nano-editor-3859002
+
+### Task 8 Update and Upgrade
+1. Update your system with `sudo apt update`. 
+
+2. Use the `sudo apt upgrade` to apply the updates downloaded and select Y for Yes. 
+
+When installing a package you use sudo apt install package_name.
+
+3. Install a specific package called tree;
```

Comment:
this displays as 1. not 3.

Reply:
I tried to change it, not sure if it worked, please let me know if it does

Reply:
Look at it yourself. What does it look like. Check it with your own eyes.

Reply:
I've had a look at it